lib/acts_as_paranoid/core.rb

```diff
@@ -112,10 +118,14 @@ def destroy
           end
         end
       else
-        destroy!
+        destroy_fully!
       end
     end
 
+    def destroy
+      destroy!
```

Comment:
This change I'm particularly weary of, it's going to be a big compatibility change in many cases.. and could break some applications.